applications/crossbar/src/modules/cb_accounts.erl

```diff
@@ -1342,8 +1342,7 @@ set_notification_preference(Context, Preference) ->
 
 -spec create_account_mod(ne_binary()) -> any().
 create_account_mod(AccountId) ->
-    Db = kz_util:format_account_mod_id(AccountId),
-    kazoo_modb:create(Db).
+    kazoo_modb:create(kazoo_modb:get_db(AccountId)).
```

Comment:
yes, but with formating account_modb there is no need to handle encoded format in modb routines, since `cb_accounts` is the only place that calls `kazoo_modb:create`, and all the maybe_create functions in kazoo_modb converts the modb to raw. I would like to be consistence anywhere to not confused in the future.